src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        Storage.storage.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
```

Comment:
Be careful with direct manipulations of `Storage.storage`. It's better to encapsulate the access to the storage. Consider creating a separate DAO (Data Access Object) class that handles the access to the storage and use it here instead of accessing the storage directly.